tool/grind.dart

```diff
@@ -0,0 +1,24 @@
+library polymer_interop.grind;
+
+import 'package:grinder/grinder.dart';
+
+const dartium = 'dartium';
```

Comment:
you can delete these it looks like, or use string interpolation below, either way